web-client/src/ducks/myRequests/actions.ts

```diff
@@ -1,15 +1,13 @@
-import { firestore } from 'firebase';
```

Comment:
Renamed createUserPost to createGeneralRequest, observeMyPostsAndResponses to observeMyRequests, and setUserPost to setGeneralRequest as part of splitting out posts into requests and offers for better understandability.

Removed Offers code.  Will add it when implement general offers.

Added return statement to dispatch - not sure how it got deleted, but it was causing functional problems when trying to add a request.